lib/rules/indent.js

```diff
@@ -1094,6 +1094,19 @@ module.exports = {
                     parameterParens.add(closingParen);
                     addElementListIndent(node.params, openingParen, closingParen, options.FunctionExpression.parameters);
                 }
+
+                if (firstToken.type === "Identifier" && firstToken.value === "async") {
+                    const openingParen = sourceCode.getTokenAfter(firstToken);
```

Comment:
Can we change this variable name? Maybe `nextToken`?

Reply:
done